### PR TITLE
TOMEE-2078 - Website improvements

### PR DESCRIPTION
**Pull request by ivanjunckes**

### CODE REVIEW

File: src/main/jbake/content/admin/index.adoc [hunk 1]
```diff
@@ -1,9 +0,0 @@
-= TomEE Administration
```
Comment (rmannibucau) [inline src/main/jbake/content/admin/index.adoc:1]: do we loose the page? If somebody bookmarked http://tomee.apache.org/admin/ is it a 404 now?

File: src/main/jbake/content/advanced/client/index.adoc [hunk 2]
```diff
@@ -1,7 +0,0 @@
-= Advanced
```
Comment (rmannibucau) [inline src/main/jbake/content/advanced/client/index.adoc:1]: same as for admin, do we loose the page?

File: src/main/jbake/content/advanced/index.adoc [hunk 3]
```diff
@@ -1,11 +0,0 @@
-= Advanced
```
Comment (rmannibucau) [inline src/main/jbake/content/advanced/index.adoc]: same ;)

File: src/main/jbake/content/advanced/jms/jms-configuration.adoc [hunk 4]
```diff
@@ -4,7 +4,7 @@
 :jbake-status: published
 :jbake-tomeepdf:
 
-== Why my ActiveMQ/JMS MDB is not scaling as expected?
+==== Why my ActiveMQ/JMS MDB is not scaling as expected?
```
Comment (rmannibucau) [inline src/main/jbake/content/advanced/jms/jms-configuration.adoc:7]: why not increasing the level linearly?

**Review by rmannibucau — COMMENTED** [review]
few points about lost index pages but overall looks a good enhancement

File: src/main/jbake/content/admin/configuration/index.adoc [hunk 5]
```diff
@@ -1,4 +1,4 @@
-= Configuration
+= How to configure
```
Comment (AndyGee) [inline src/main/jbake/content/admin/configuration/index.adoc]: Server Configuration or How to Configure the Server

File: src/main/jbake/content/admin/index.adoc (hunk 1 above)
Comment (AndyGee) [inline src/main/jbake/content/admin/index.adoc:1]: Looking at most server sites either 'Docs' or 'Documentation' is consistent with what people expect

File: src/main/jbake/content/advanced/client/index.adoc (hunk 2 above)
Comment (AndyGee) [reply src/main/jbake/content/advanced/client/index.adoc:1]: Same as above - It makes sense to hit documentation from one place

File: src/main/jbake/content/advanced/client/jndi.adoc [hunk 6]
```diff
@@ -1,4 +1,4 @@
-= Advanced
+= JNDI
```
Comment (AndyGee) [inline src/main/jbake/content/advanced/client/jndi.adoc]: Java Naming and Directory Interface (JNDI)

File: src/main/jbake/content/advanced/index.adoc (hunk 3 above)
Comment (AndyGee) [reply src/main/jbake/content/advanced/index.adoc]: Also the same. I think that the site is pretty new and if linked then it's an easy change.

File: src/main/jbake/content/advanced/jms/jms-configuration.adoc (hunk 4 above)
Comment (AndyGee) [inline src/main/jbake/content/advanced/jms/jms-configuration.adoc:7]: Why **is** my ActiveMQ/JMS MDB not scaling as expected?

File: src/main/jbake/content/advanced/jms/jms-configuration.adoc [hunk 7]
```diff
@@ -61,7 +61,7 @@ public static class MyMdb implements MessageListener {
 </Resource>
 ----
 
-== Slow consumption
+==== Slow consumption
```
Comment (AndyGee) [inline src/main/jbake/content/advanced/jms/jms-configuration.adoc]: Slow Message Consumption

File: src/main/jbake/content/advanced/shading/index.adoc [hunk 8]
```diff
@@ -1,11 +1,9 @@
-= TomEE Shading
+= Fat / Uber jars - Shade Plugin
```
Comment (AndyGee) [inline src/main/jbake/content/advanced/shading/index.adoc]: Fat / Uber jars - Using the Shade Plugin

File: src/main/jbake/content/community/index.adoc [hunk 9]
```diff
@@ -3,7 +3,7 @@
 :jbake-type: page
 :jbake-status: published
 
-=== In code veritas est
+=== Source Code
```
Comment (AndyGee) [inline src/main/jbake/content/community/index.adoc]: The Source Code

File: src/main/jbake/content/community/index.adoc [hunk 10]
```diff
@@ -19,7 +19,7 @@ Facebook, Twitter, Google+: if that speaks to you just follow this link:social.h
 
 For more information about available support for TomEE please have a look on link:../security/support.html[Support] page.
 
-=== Contribute the this website
+=== Contribute to this website
```
Comment (AndyGee) [inline src/main/jbake/content/community/index.adoc]: Contribute to This Website

File: src/main/jbake/content/developer/classloading/index.adoc [hunk 11]
```diff
@@ -1,4 +1,4 @@
-= TomEE ClassLoading
+= TomEE Classloading
```
Comment (AndyGee) [inline src/main/jbake/content/developer/classloading/index.adoc]: The TomEE ClassLoader or Understanding The TomEE ClassLoader

File: src/main/jbake/content/developer/configuration/cxf.adoc [hunk 12]
```diff
@@ -1,10 +1,10 @@
-= TomEE and CXF Configuration
+= CXF Configuration - JAX-RS and JAX-WS
 :jbake-date: 2016-03-16
 :jbake-type: page
 :jbake-status: published
 :jbake-tomeepdf:
 
-TomEE relies on Apache CXF for JAX-WS and JAX-RS, it doesn't provides all CXF modules but most comomn
+TomEE relies on Apache CXF for JAX-RS and JAX-WS, it doesn't provides all CXF modules but most common
```
Comment (AndyGee) [inline src/main/jbake/content/developer/configuration/cxf.adoc]: TomEE relies on Apache CXF for JAX-RS (RESTful Services) and JAX-WS (Web Services). It does not provide all CXF modules, but the most common

File: src/main/jbake/content/developer/configuration/cxf.adoc [hunk 13]
```diff
@@ -1,10 +1,10 @@
-= TomEE and CXF Configuration
+= CXF Configuration - JAX-RS and JAX-WS
```
Comment (AndyGee) [inline src/main/jbake/content/developer/configuration/cxf.adoc]: CXF Configuration - JAX-RS (RESTful Services) and JAX-WS (Web Services)

File: src/main/jbake/content/developer/ide/index.adoc [hunk 14]
```diff
@@ -1,10 +1,10 @@
-= TomEE and IDE
+= IDEs
```
Comment (AndyGee) [inline src/main/jbake/content/developer/ide/index.adoc]: Integrated Development Environments (IDEs)

File: src/main/jbake/content/developer/json/index.adoc [hunk 15]
```diff
@@ -1,4 +1,4 @@
-= TomEE and JSON
+= TomEE 7 and Apache Johnzon
```
Comment (AndyGee) [inline src/main/jbake/content/developer/json/index.adoc]: TomEE and Apache Johnzon - JAX-RS JSON Provider

File: src/main/jbake/content/developer/testing/applicationcomposer/index.adoc [hunk 16]
```diff
@@ -1,4 +1,4 @@
-= ApplicationComposer: The TomEE Swiss Knife
+= ApplicationComposer: the swiss knife of TomEE
```
Comment (AndyGee) [inline src/main/jbake/content/developer/testing/applicationcomposer/index.adoc]: ApplicationComposer: The TomEE Swiss Knife

File: src/main/jbake/content/developer/testing/index.adoc [hunk 17]
```diff
@@ -1,4 +1,4 @@
-= TomEE and Testing
+= Testing
```
Comment (AndyGee) [inline src/main/jbake/content/developer/testing/index.adoc]: Unit Testing

File: src/main/jbake/content/developer/tools/index.adoc [hunk 18]
```diff
@@ -1,4 +1,4 @@
-= TomEE developer tools
+= Build tools
```
Comment (AndyGee) [inline src/main/jbake/content/developer/tools/index.adoc]: Build Tools and Plugins

File: src/main/jbake/content/documentation.adoc [hunk 19]
```diff
@@ -0,0 +1,26 @@
+= TomEE Documentation
+:jbake-date: 2016-03-16
+:jbake-type: page
+:jbake-status: published
+:jbake-tomeepdf:
+
+=== Administration
+- link:admin/configuration/index.html[How to configure]
```
Comment (AndyGee) [inline src/main/jbake/content/documentation.adoc]: Server Configuration

File: src/main/jbake/content/documentation.adoc [hunk 20]
```diff
@@ -0,0 +1,26 @@
...
+- link:admin/directory-structure.html[Directory Structure]
+- link:admin/cluster/index.html[Clustering]
```
Comment (AndyGee) [inline src/main/jbake/content/documentation.adoc]: Clustering and High Availability (HA)

File: src/main/jbake/content/documentation.adoc [hunk 21]
```diff
@@ -0,0 +1,26 @@
...
+
+=== Developers
+- link:developer/ide/index.html[IDEs - Eclipse, Intellij Idea and Netbeans]
+- link:developer/testing/index.html[Testing - Arquillian, OpenEJB JUnit, TomEE Embedded and ApplicationComposer]
```
Comment (AndyGee) [inline src/main/jbake/content/documentation.adoc]: Unit Testing - Arquillian, OpenEJB JUnit, TomEE Embedded and ApplicationComposer

File: src/main/jbake/content/documentation.adoc [hunk 22]
```diff
@@ -0,0 +1,26 @@
...
+- link:developer/tools/index.html[Build Tools - Maven and Gradle]
+- link:developer/migration/tomee-1-to-7.html[Migrating from TomEE 1 to 7]
```
Comment (AndyGee) [inline src/main/jbake/content/documentation.adoc]: Migrating From TomEE 1.x to 7.x

File: src/main/jbake/content/documentation.adoc [hunk 23]
```diff
@@ -0,0 +1,26 @@
...
+- link:developer/json/index.html[TomEE 7 and Apache Johnzon (JSON Mapper)]
+- link:developer/configuration/cxf.html[Apache CXF Configuration - JAX-RS and JAX-WS]
```
Comment (AndyGee) [inline src/main/jbake/content/documentation.adoc]: Apache CXF Configuration - JAX-RS (RESTful Services) and JAX-WS (Web Services)

File: src/main/jbake/content/documentation.adoc [hunk 24]
```diff
@@ -0,0 +1,26 @@
...
+- link:developer/classloading/index.html[TomEE Classloading]
```
Comment (AndyGee) [inline src/main/jbake/content/documentation.adoc]: Understanding the TomEE ClassLoader

File: src/main/jbake/content/documentation.adoc [hunk 25]
```diff
@@ -0,0 +1,26 @@
...
+
+=== Advanced
+- link:advanced/applicationcomposer/index.html[`ApplicationComposer` with JBatch]
+- link:advanced/setup/index.html[How to setup TomEE in production]
```
Comment (AndyGee) [inline src/main/jbake/content/documentation.adoc]: How to Setup TomEE in Production

File: src/main/jbake/content/documentation.adoc [hunk 26]
```diff
@@ -0,0 +1,26 @@
...
+- link:advanced/shading/index.html[Fat / Uber jars - Shade Plugin]
```
Comment (AndyGee) [inline src/main/jbake/content/documentation.adoc]: Fat / Uber Jar Deployment using the Maven Shade Plugin

**Review by AndyGee — COMMENTED** [review]
Overall I think that docs should be under one link, which is what users have come to expect. So I like this remodelling, as the previous version was confusing.

**Pull request merged**